include/Standalone/Dialect/Tpp/TppOps.td

```diff
@@ -25,6 +25,7 @@ class StaticMemRefRankOf<list<Type> allowedTypes, list<int> ranks> :
 def TppMemRef : StaticMemRefRankOf<[AnyFloat], [1, 2]>;
 def TppPackedMemrefInput : StaticMemRefRankOf<[AnyFloat], [1, 2, 3]>;
 def TppBRGEMMemrefInput : StaticMemRefRankOf<[AnyFloat], [3]>;
+def TppBRGEMMPackedMemrefInput : StaticMemRefRankOf<[AnyFloat], [3,4]>;
```

Comment:
Also IREE direction for handling bf16 is maybe a good idea here. We are opening up this op too much. But let's discuss later and ignore this comment for now.